### ISSUE TITLE
SDL2 path incorrect

### ISSUE THREAD

**Issue by Shambler2** [PCemOnMac/PCemV17macOS#15]
Hi All,

Very minor issue that I ran into while compiling the latest main on an M1 Mac. 

System: Macbook Pro M1
Pre-requisites installed through ARM native version of homebrew (sdl2)
wxmac compiled as per instructions

Compile fails due to SDL.h (and a few other related header files) not being in the correct path. 

Worked around it by swapping out all angle bracketed header includes that contain  <SDL2/*> for double quoted includes e.g. <SDL2/sdl.h> to "SDL.h". There was roughly 10 edits I had to make then it compiled fine, all were in files that interacted with wx-mac.

Not sure if anyone else came across this or its something down to the way I have my system set up.

**Comment by Shambler2** [comment]
Opened pull request #16 with the changes I made to make it compile on my system - feel free to close this & the pull request if it causes issues anywhere else

**Comment by kyr0** [comment]
Hi @Shambler2 

thank you for the PR! Are you running Big Sur? I also commented on your PR.

**Comment by Shambler2** [comment]
Hi @kyr0 yes I'm running Big Sur (should have mentioned that, whoops).

From what I can see yes that may well be the case - It's unlikely the approach I suggested is the right one but wanted to put it out there anyway as a workaround for anyone who might run into the same issue I did but not know how to fix it.

I'd be keen to understand if this is something anyone else has run into whilst testing this though!
